docs/includes/net-standard-table.md

```diff
@@ -0,0 +1,19 @@
+| Platform Name              | Alias       |        |        |        |        |        |        |        |       |
+|----------------------------|-------------|--------|--------|--------|--------|--------|--------|--------|-------|
+| .NET Standard              | netstandard | [1.0]  | [1.1]  | [1.2]  | [1.3]  | [1.4]  | [1.5]  | [1.6]  | [2.0] |
+| .NET Core                  | netcoreapp  | &rarr; | &rarr; | &rarr; | &rarr; | &rarr; | &rarr; | 1.0    | 2.0   |
+| .NET Framework             | net         | &rarr; | 4.5    | 4.5.1  | 4.6    | &rarr; | &rarr; | &rarr; | 4.6.1 |
+| Mono/Xamarin Platforms     |             | &rarr; | &rarr; | &rarr; | &rarr; | &rarr; | &rarr; | &rarr; | vNext |
+| Universal Windows Platform | uap         | &rarr; | &rarr; | &rarr; | &rarr; | 10.0   | &rarr; | &rarr; | vNext |
```

Comment:
@weshaggard the information on how to read the table is in the .NET Standard topic itself. Since this information was reused in other topics, I moved to this "include" file so it can be reused in several topics if needed. But I'll change the instructions a bit. New commit coming soon.